References: bsc#1204085
Running yast2 services-manager and enabling the service
at boot time was not working.

--- spice-vdagent-0.22.1/data/spice-vdagentd.service.orig	2022-10-10 14:43:46.164921761 -0600
+++ spice-vdagent-0.22.1/data/spice-vdagentd.service	2022-10-10 14:49:48.988930407 -0600
@@ -24,3 +24,4 @@ Restart=on-failure
 
 [Install]
 Also=spice-vdagentd.socket
+WantedBy=multi-user.target
